test/metagraphs.jl

```diff
@@ -78,6 +78,32 @@ import Base64:
         U = @inferred(weighttype(mg))
         @test @inferred(nv(MetaGraph{T,U}(6))) == 6
 
+        # get_prop with default argument
+        # vertices
+        set_prop!(mg, nv(mg), :testprop, "exists")
+        @test get_prop(mg, nv(mg), :testprop, "nonexistent") == "exists"
+        @test get_prop(mg, nv(mg), :testprop_nonexist, "nonexistent") == "nonexistent"
+        @test get_prop(mg, nv(mg) + 100, :testprop_nonexist, "nonexistent") == "nonexistent"
```

Comment:
I'd say we still want this to error?

Reply:
You mean trying to get a prop from an edge or vertex that does not exist? I think you are right. Added a check for that.